src/pages/Repos/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect} from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+import { RepoCard, RepoModal, SearchForm, BackButton } from '../../components'
+import './style.css'
+
+const Repos = () => {
+    const { username } = useParams();
+    const [repoInfo, setRepoInfo] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [modal, setModal] = useState(false);
+    const [modalData, setModalData] = useState([]);
+    const [filteredRepos, setFilteredRepos] = useState([]);
+
+    useEffect( async ()=>{
+        const { data } = await axios.get(`https://api.github.com/users/${username}/repos`);
+        setRepoInfo(data);
+        setFilteredRepos(data);
+        setLoading(false);
+    }, [])
+
+    function closeModal(){
```

Comment:
Cool that you used Modals, we just redirected to another page which was much less elegant